### ISSUE TITLE
Backport fixes for HTTPS and WMS issues of gh-pages to master

### ISSUE THREAD

**Issue by chrismayer** [geoext/geoext2#372]
Due to  'Mixed Content' issues when opening the online examples with HTTPS and some broken WMS references in the online examples several fixes have been made in the gh-pages branch to make the online examples work again (see #359 and #371).

These fixes should be ported to the master branch as well, so they won't be lost on the next update of the gh-pages branch. The following commits should be the base for the backport:

  - c8e1d7fc7056f4184713d14a0275878ebb587e3b
  - 14cde9eb1bb192ed5befb1343d3d541370dbd0bc
  - 01a6cccad5adc5a4dee86c2c83bb68e1830672d6
  - ea0347fa4b5d9ded84c201c52e7bb67665d545b6
  - e3275ace9c76b50c3ccec81e8e53b5a55ca486d9

**Comment by marcjansen** [comment]
Do we still need to do this?

**Comment by chrismayer** [comment]
I think it is an important reminder for the backport of the HTTPS stuff in gh-pages branch.
Doing the backport itself would be the better option of course :wink: Someone volunteering?

**Comment by marcjansen** [comment]
Yeah, I wasn't sure if we had done it. So this is still open. I'd prefer someone else doing it 😇

**Comment by bentrm** [comment]
Would it be ok to load OpenLayers from a CDN that supports https? This way, OpenLayers can be removed in the gh-pages branch and will be available executing the examples locally. Thinking about [cdnjs](https://cdnjs.com/libraries/openlayers).

**Comment by marcjansen** [comment]
Yes that would be ok

**Comment by chrismayer** [comment]
OK for me.
